src/Models/BaseElement.php

```diff
@@ -470,6 +480,33 @@ public function Top()
         return (Controller::has_curr()) ? Controller::curr() : null;
     }
 
+    /**
+     * Determines whether this elemental block is indexable in search.
+     *
+     * By default, this uses the configurable variable is_indexable, but
+     * this method can be overridden to provide more complex logic if required.
+     *
+     * @return boolean
+     */
+    public function getIsIndexable(): bool
+    {
+        return (bool)$this->config()->get('is_indexable', Config::UNINHERITED);
+    }
+
+    /**
+     * Provides content to be indexed in search.
+     *
+     * @return string|null HTML
```

Comment:
This is intentionally nullable so that developers can return null from an overridden method in a subclass if they explicitly don't want content to be indexed. Let me know if you're okay with that approach - I should probably specify that in the PHPDoc for clarity.

Reply:
What's the functional difference between `null` and `''` (empty string) though?  Is the something in the FullTextSearch module (or another search module?) that explicitly tests for `null`?

Reply:
It's a semantic difference rather than a functional one so I'm not too worried about it at the end of the day.

Reply:
I've left this nullable for now and made the typehint nullable because it is quite common for developers to return null from methods such as this one in ss projects - happy to remove nullability in the end if you want, but I had to make a call.

Reply:
Returning a blank string also won't get indexed :-) i.e. what search term could and end user put into a search box to match blank?  I just can't see a use-case where `null` does something that `''` doesn't, so I'd prefer to keep the API simple and just have a `string` return type

Reply:
I can agree to that logic! I'll make the change.